src/app/core/learning-object-module/submissions/submissions.routes.ts

```diff
@@ -24,13 +24,10 @@ export const SUBMISSION_ROUTES = {
     },
     /**
      * Request to cancel a submission
-     * @param userId - The id of the author of the learning object
      * @param learningObjectId - The id of the learning object to cancel the submission for
      * @auth required
      */
-    DELETE_SUBMISSION(params: { userId: string, learningObjectId: string }) {
-        return `${environment.apiURL}/users/${encodeURIComponent(
-            params.userId
-        )}/learning-objects/${encodeURIComponent(params.learningObjectId)}/submissions`;
+    DELETE_SUBMISSION(params: { learningObjectId: string }) {
```

Comment:
```suggestion
    DELETE_SUBMISSION(learningObjectId: string) {
```
Please remove the params object. All references to DELETE_SUBMISSION will need to updated to only pas the learningObjectId

Reply:
+1